### PR TITLE
Improve some features of the Map

### PR DESCRIPTION
**Pull request by manuelsanchez2**
In this pull request, I improved some aspects regarding the visibility of the spots on the map. 

Screenshot from 2020-10-03 15-34-10

### CODE REVIEW

File: client/src/components/MainMap/MainMap.js [hunk 1]
```diff
@@ -5,6 +5,27 @@ const MainMap = styled.main`
   flex: 1;
   overflow: hidden;
   position: relative;
+
+  .mapboxgl-ctrl-geocoder {
+    width: 85%;
+  }
+
+  .mapboxgl-ctrl-geocoder--input {
+  }
```
Comment (LebkuchenCJ) [inline client/src/components/MainMap/MainMap.js]: Do you still need this?

File: client/src/components/MainMap/MainMap.js [hunk 2]
```diff
@@ -5,6 +5,27 @@ const MainMap = styled.main`
...
+  .mapboxgl-popup-content {
+    border-radius: 10px;
+    box-shadow: 0.5px 0px 20px 0px rgba(0, 0, 0, 0.3);
+  }
+
+  .mapboxgl-popup-close-button {
+    right: 6px;
+    top: 5px;
+    background-color: red;
+    color: white;
```
Comment (LebkuchenCJ) [inline client/src/components/MainMap/MainMap.js]: You could think of moving these into your global styles

File: client/src/components/Spot/Spot.js [hunk 3]
```diff
@@ -6,6 +6,31 @@ import mapMarkerRed from "../../assets/icons/map-marker-red.svg";
...
+
+const SpotPopUpCloseButton = styled.button`
+  background: red;
+  border: 0;
+  width: 21px;
+  color: white;
+  height: 21px;
+  position: absolute;
+  top: 4px;
+  right: 4px;
```
Comment (LebkuchenCJ) [inline client/src/components/Spot/Spot.js:19]: These seem to be quite exact. How do they behave on responsiveness?

**Review by LebkuchenCJ — COMMENTED** [review]
GJ 👍 good to go
Only small comments 💯

File: client/src/components/MainMap/MainMap.js (hunk 1 above)
Comment (manuelsanchez2) [reply client/src/components/MainMap/MainMap.js]: Ups forgot to delete! Thanks!!!

File: client/src/components/MainMap/MainMap.js (hunk 2 above)
Comment (manuelsanchez2) [reply client/src/components/MainMap/MainMap.js]: Completely true!

File: client/src/components/Spot/Spot.js (hunk 3 above)
Comment (manuelsanchez2) [reply client/src/components/Spot/Spot.js:19]: It works relatively okay because the container's height is always the same

**Pull request merged**